wagtail/core/tests/test_audit_log.py

```diff
@@ -24,19 +24,19 @@ def setUp(self):
     def test_register_log_actions_hook(self):
         # testapp/wagtail_hooks.py defines a 'blockquote' rich text feature with a hallo.js
```

Comment:
I don't think this comment applies here, unless I'm missing some rich text integration...

Reply:
This was already in the old code in wagtail/admin/tests/test_audit_log.py.